doc/dev_guide/release_management.rst

```diff
@@ -10,92 +10,134 @@ Release management
 Release policy
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-A Tarantool release is identified by three digits, for example, 1.7.7.
-We use these digits according to their definitions provided at http://semver.org:
+A Tarantool release is identified by three digits, for example, 1.10.7:
+
+* The first digit stands for a MAJOR release that introduces
+  some *major changes*. Up to now, there has been only one major release jump
+  when we delivered the 2.x release series with the SQL support.
+* The second digit stands for a MINOR release that is used for
+  introducing new *features*. Some of the features may have no backward
+  compatibility.
+* The third digit is for PATCH releases that contain only
+  backward-compatible *bug fixes*.
+
+In the PATCH digit, we reflect how stable a release is:
+
+* ``0`` meaning **alpha**
+* ``1`` meaning **beta**
+* ``2`` and above meaning **stable**.
+
+So, each MINOR release series goes through a development-maturity life cycle
+as follows:
+
+1. **Alpha**. Once a quarter, we start off with a new alpha version,
+   such as 2.3.0, 2.4.0, and so on. It is not what an alpha release usually
+   means in the software release life cycle but rather the current trunk version
+   which is under heavy development and can be unstable.
+   The current alpha version always lives in the master branch.
+
+2. **Beta**. When all the features planned are implemented, we fork a new branch
+   from the master branch and tag it as a new beta version.
+   It contains ``1`` for the PATCH digit, e.g., 2.3.1, 2.4.1, and so on.
+   This version is still unstable although there're no critical regressions
+   in it since the last stable release. We do this quarterly as well.
```

Comment:
I would move accents a bit:

* From 'is unstable' to 'is not proven by time to be stable (a feature set is just frozen)'.
* From 'no critical regressions' to 'no known critical regressions'.

Like so:

> This version is not proven by time to be stable (a feature set is just frozen) although there're no known critical regressions in it since the last stable release.